src/routes/github/github-oauth-router.ts

```diff
@@ -11,14 +11,13 @@ import { createHashWithSharedSecret } from "utils/encryption";
 
 const logger = getLogger("github-oauth");
 const appUrl = envVars.APP_URL;
-const scopes = ["user", "repo"];
```

Comment:
Hmm, actually sorry just realise that, @joshkay10 do you mind have a look at this as well? I am not sure will the `create-branch` require any scope from this user token as well?

Reply:
We will need repo still, but not user.

Reply:
Thanks, @joshkay10. Just adding here for our future reference:

> Access tokens require the repo scope for private repositories.
https://docs.github.com/en/enterprise-cloud@latest/graphql/guides/using-the-graphql-api-for-discussions